### ISSUE TITLE
Groundtruth creation

### ISSUE THREAD

**Issue by betatim** [betatim/dreamview#5]
Currently we use a majority vote of mapswipe users to decide the ground-truth of a tile.

From looking at some tiles which we classify as "boring" in the current setup I noticed a few contain things that should be mapped. The mapswipe user votes might also be noisy.

How to improve the ground truth?

One idea is to use [OSM QA tiles](https://osmlab.github.io/osm-qa-tiles/). If the corresponding QA tile contains something this is a tile that is "interesting", if not we assume it is empty because it is boring (not because it hasn't been mapped yet).

Next step: download a country extract of QA tiles and see how easy it is to determine if a tile is "empty"

**Comment by betatim** [comment]
Another approach could be to use the rendered OSM tiles and and do pattern matching to decide if they are "empty" or not.

A further advantage of using QA tiles or OSM tiles is that we are not limited to training data that has been mapswiped.

**Comment by betatim** [comment]
https://github.com/tilezen/mapbox-vector-tile for reading QA tiles in python
